app/helpers/application_helper.rb

```diff
@@ -29,7 +29,7 @@ def cart_properties
   end
 
   def options_for_status
-    ['Completed', 'Cancelled', 'Pending']
+    ['Completed', 'Cancelled', 'Pending', 'Delivered']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Use `%w` or `%W` for array of words.